aspnetcore/migration/22-to-30.md

```diff
@@ -1216,6 +1216,7 @@ Review breaking changes:
 * [Complete list of breaking changes in the ASP.NET Core 3.0 release](https://github.com/aspnet/Announcements/issues?page=2&q=is%3Aissue+is%3Aopen+label%3A%22Breaking+change%22+label%3A3.0.0)
 * [Breaking API changes in Antiforgery, CORS, Diagnostics, MVC, and Routing](https://github.com/aspnet/Announcements/issues/387). This list includes breaking changes for compatibility switches.
 * For a summary of 2.2-to-3.0 breaking changes across .NET Core, ASP.NET Core, and Entity Framework Core, see [Breaking changes for migration from version 2.2 to 3.0](/dotnet/core/compatibility/2.2-3.0).
+* Some Cookie SameSite defaults changed to `None`.  For more information, see [this](https://github.com/aspnet/Announcements/issues/348) GitHub announcement.
```

Comment:
```suggestion
* [Some Cookie SameSite defaults changed to None (aspnet/Announcements #348)](https://github.com/aspnet/Announcements/issues/348).
```

Period present because other lines have one, but seems like ... this is just an unimportant nit ... the whole list could be just links really (i.e., no periods and no additional text).